public/themes.txt

```diff
@@ -72,7 +72,7 @@
 #https://github.com/ianfab/fairyground/issues
 
 
-*|lettertiles,letters,userdefined|
+*|lettertiles,letters,meridaletters,cburnettletters,userdefined|

```

Comment:
I agree that now that we have a dropdown it doesn't hurt to add all of them. Personally I would likely put those two before the other two letter sets since they look nicer to me, but probably doesn't matter much either way.

Reply:
I will change the order.